source/documentation/modules/math.html.md.erb

```diff
@@ -0,0 +1,494 @@
+---
+title: sass:math
+---
+
+<%= partial '../snippets/built-in-module-status' %>
+
+## Variables
+
+<% function 'math.$e' do %>
+  Equal to the value of the mathematical constant `e`.
```

Comment:
```suggestion
  Equal to the value of the mathematical constant *e*.
```

Since you're referring to a mathematical constant here rather than programming language syntax, it's probably a good idea to use a different typographic convention. Italics are often used when math is included in prose.

Reply:
Done.